packages/react/src/components/Menu/Menu.tsx

```diff
@@ -164,6 +175,9 @@ class Menu extends AutoControlledComponent<WithAsProp<MenuProps>, MenuState> {
     const itemsCount = _.filter(items, item => getKindProp(item, 'item') !== 'divider').length
     let itemPosition = 0
 
+    const itemOverridesFn = this.handleItemOverrides(variables)
```

Comment:
nit: may suggest to rename these to something like `overrideItemProps` and `overrideDividerProps` - think that it will make code easier to read